packages/itwin/tree-widget/src/components/trees/VisibilityTreeRenderer.tsx

```diff
@@ -30,32 +61,41 @@ export const useVisibilityTreeRenderer = (iconsEnabled: boolean, descriptionsEna
 };
 
 const imageLoader = new TreeImageLoader();
+
 /**
  * Creates node renderer which renders node with eye checkbox.
  * @public
  */
-export const createVisibilityTreeNodeRenderer = (iconsEnabled: boolean, descriptionEnabled: boolean) => {
-  return (props: TreeNodeRendererProps) => ( // eslint-disable-line react/display-name
-    <TreeNodeRenderer
-      {...props}
-      checkboxRenderer={visibilityTreeNodeCheckboxRenderer}
-      descriptionEnabled={descriptionEnabled}
-      imageLoader={iconsEnabled ? imageLoader : undefined}
-      className={classNames("with-checkbox", props.className)}
-    />
-  );
+export const createVisibilityTreeNodeRenderer = ({ levelOffset = 20, expansionToggleWidth = 24, disableRootNodeCollapse = false, descriptionEnabled, iconsEnabled }: VisibilityTreeRendererProps) => {
+  return function VisibilityTreeNodeRenderer(treeNodeProps: TreeNodeRendererProps) {
+    const nodeOffset = treeNodeProps.node.depth * levelOffset + (treeNodeProps.node.numChildren === 0 ? expansionToggleWidth : 0);
+    return (
+      <TreeNodeRenderer
+        {...treeNodeProps}
+        node={{ ...treeNodeProps.node, depth: 0, numChildren: 1 }}
```

Comment:
Can you add comment explaining why `depth` and `numChildren` are overridden?

Reply:
fixed